src/routes/home/components/AnnouncementDialog.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <BaseDialog
+    id="announcement-dialog"
+    v-model="show"
+    variant="light"
+    :opacity="1"
+    data-cy="announcement-dialog"
+    scrollable
+    :persistent="false"
+    :max-width="750"
+    :title="announcementData.title"
+  >
+    <template #activator>
+      <v-btn
+        color="newSecondary"
+        rounded="0"
+        block
+      >
+        {{ announcementData.activatorText }}
+      </v-btn>
+    </template>
+    <template #body>
+      <div v-if="announcementData.displayCards.length" class="d-flex justify-center my-4">
+        <GameCard
+          v-for="card in announcementData.displayCards"
+          :key="card.suit + card.rank"
+          :suit="card.suit"
+          :rank="card.rank"
+          class="card mb-10"
+          :high-elevation="true"
+        />
+      </div>
+      <div v-if="announcementData.imgSrc" class="d-flex justify-center">
+        <img class="w-50" :src="announcementData.imgSrc">
+      </div>
+      <div v-for="(text,i) in announcementData.announcementText" :key="i">  
+        <h2 v-if="text.heading">
+          {{ text.heading }}
+        </h2>  
+        <p> {{ text.paragraph }} </p>  
+      </div>  
+    </template>
+    <template #actions>
+      <v-btn
+        data-cy="announcement-dialog-close"
+        color="surface-1"
+        variant="flat"
+        @click="close"
+      >
+        {{ t('global.close') }}
+      </v-btn>
+    </template>
+  </BaseDialog>
+</template>
+
+<script setup>
+import BaseDialog from '@/components/BaseDialog.vue';
+import { ref, onMounted } from 'vue';
+import { useI18n } from 'vue-i18n';
+import GameCard from '@/routes/game/components/GameCard.vue';
+import { getLocalStorage, setLocalStorage } from '_/utils/local-storage-utils.js';
+import { announcementData } from '../../../data/announcementData';
+
+const { t } = useI18n();
+const show = ref(false);
+const preferenceSaved= ref(false);
+
+const close = () => {
+  if (!preferenceSaved.value) {
+    setLocalStorage('announcement', announcementData.id);
+  }
+  show.value = false;
+  preferenceSaved.value = true;
+};
+
+onMounted(() => {
+  if (getLocalStorage('announcement') !== announcementData.id) {
+    show.value = true;
+  }
+});
+</script>
+
+<style scoped>
+  .card:nth-child(n+2){
+    margin-left: -9% !important;
+  }
+
+
```

Comment:
There is some weirdness around the height of the images on mobile. It boils down to a discrepancy created by the css applied to the image's direct container inside `GameCard.vue`, which sets the `max-height: 10vh` and also sets the width based on a calculation. The image itself winds up taking up a different amount of space than its container, which looks weird and also messes up the margin here:

<img width="498" alt="image" src="https://github.com/user-attachments/assets/46a09d06-f36e-48e7-97ba-aa22dd2bc607">


I don't want to mess with the `GameCard` css in this PR because it could have sweeping implications throughout the GameView. So rather than tackle that here, let's hack our way around it with a `:deep()` selector that overrides the `max-height` like so
```suggestion
  :deep(.player-card) {
    max-height: inherit;
  }
```

This gives us the following:
<img width="499" alt="image" src="https://github.com/user-attachments/assets/522bed96-0a06-46ff-8afd-a456c462b50e">


From there we can investigate the larger issue with the GameCard css in its own issue